client/consul_template.go

```diff
@@ -430,11 +424,32 @@ func parseTemplateConfigs(tmpls []*structs.Template, taskDir string,
 }
 
 // runnerConfig returns a consul-template runner configuration, setting the
-// Vault and Consul configurations based on the clients configs.
-func runnerConfig(config *config.Config, vaultToken string) (*ctconf.Config, error) {
+// Vault and Consul configurations based on the clients configs. The parameters
+// are the client config, Vault token if set and the mapping of consul-templates
+// to Nomad templates.
+func runnerConfig(config *config.Config, vaultToken string,
```

Comment:
Add a verb like newRunnerConfig or makeRunnerConfig to make it more distinct from the variable it's assigned to above.